Sem. 09/Pract.10/Task14.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+using namespace std;
+constexpr int MAX=100000;
+void getDigitOccurences (const char* digit, char* result){
+    if(!digit || !result) return;
+
+    int counter[10]={0};
+    for (const char* ptr = digit; *ptr; ptr++){
+        if(*ptr>='0'&& *ptr<='9'){
+            counter[*ptr-'0']++;
+        }
+    }
+
+    for(int i=0; i<10; i++){
+        if(counter[i]>0){
+        int count=counter[i];
+        if(count>=10){
+            *result=(count/10)+'0';
+            result++;
+         }
+         *result=(count%10) + '0';
+         result++;
+         *result= 'x';
+         result++;
+         *result= i+'0';//cifrata
+         result++;
+         *result=',';
+         result++;
+    } 
+}
```

Comment:
Please make a function to convert a digit into a character symbol of the digit and backward. This will improve code readability by separating core logic from details.
`'1' => 1`
`1 => '1'`